doc/source/tutorials/lab.rst

```diff
@@ -299,12 +342,52 @@ a two-qubit system:
                 threshold: 0.0002694329123116206
                 iq_angle: 4.912447775569025
 
+And in the case of having a chip with coupler qubits
+we need to following changes to the previous runcard:
+
+.. code-block::  yaml
+
+    qubits: [0, 1]
+    couplers: [0]
+
+    topology: {0: [0, 1]}
+
+    native_gates:
+        two_qubit:
+            0-1:
+                CZ:
+                - duration: 30
+                amplitude: 0.6025
+                shape: Rectangular()
+                qubit: 2
+                relative_start: 0
+                type: qf
+
+                - type: virtual_z
+                phase: -1
+                qubit: 1
+                - type: virtual_z
+                phase: -3
+                qubit: 2
+
+                - type: coupler
+                phase: -3
```

Comment:
```suggestion
```
I think here there is no phase for these pulses. I got an error when I try to create the platform. If we need a phase maybe something is missing from the code.